helm_deploy/create-and-vary-a-licence-api/values.yaml

```diff
@@ -47,6 +47,7 @@ generic-service:
       NOTIFY_TEMPLATES_DATESCHANGED: "DATES_CHANGED"
       NOTIFY_TEMPLATES_VARIATIONAPPROVED: "VARIATION_APPROVED"
       NOTIFY_TEMPLATES_VARIATIONREFERRED: "VARIATION_REFERRED"
+      NOTIFY_TEMPLATES_UNAPPROVEDLICENCE: "UNAPPROVED_LICENCE"
```

Comment:
did you work out what these were being set for?

Reply:
yes